### ISSUE TITLE
Steam Flatpak

### ISSUE THREAD

**Issue by QueDemais** [flathub/com.valvesoftware.Steam#1169]
Hello,

After installing, do I need to do any extra configuration to be functional 100% in the distribution ?
Example: Fedora Kinoite

Will Valve give the verified seal too?

Thanks 👍😄

**Comment by doogie544** [comment]
The flatpak is not an official package by valve so it can't have the verified seal.  Valve devs have helped get it working in the past when there were issues but unless they take over the package and make it official it can't have the verified seal/checkmark if I understand it correctly.

I think many of use are hoping that valve will adopt it and  make the flatpak an officially supported version but so far that has not happened

**Comment by user1-github** [comment]
> The flatpak is not an official package by valve so it can't have the verified seal. Valve devs have helped get it working in the past when there were issues but unless they take over the package and make it official it can't have the verified seal/checkmark if I understand it correctly.
> 
> I think many of use are hoping that valve will adopt it and make the flatpak an officially supported version but so far that has not happened

But I've read that in order to get the "verified" seal, the app doesn't necessarily have to be packaged directly by the developer. It can also be verified if the developer approves a third party to package his app. For example, when it comes to very big projects like Libreoffice, it's obvious that it can't be packaged directly by the developers, so different maintainers are responsible for the various package formats. And it seems to me the maintainers who package the Libreoffice Flatpak aren't even affiliated with The Document Foundation, but Libreoffice is still verified in Flathub.

So I say maybe if someone will try to contact Valve and ask if it for Flathub verification approval, I think there is a chance it will accept that.

**Comment by herp1derp2** [comment]
The only reason I am looking at Steam's flatpak version is because of Gnome. There is no tray icon or entry in its "background app" in quick setting for official Steam application. I really hope either Gnome or Steam developers fix this issue.

**Comment by Roman19S** [comment]
> The only reason I am looking at Steam's flatpak version is because of Gnome. There is no tray icon or entry in its "background app" in quick setting for official Steam application. I really hope either Gnome or Steam developers fix this issue.

It's not about the package, you need an extension called AppIndicator and KStatusNotifierItem Support. Gnome does not support tray icons in any version right now. You can get the extension from the Gnome website: https://extensions.gnome.org/extension/615/appindicator-support/ , or install it using the extension manager flatpak https://flathub.org/apps/com.mattjakeman.ExtensionManager , I recommend the latter.

**Comment by herp1derp2** [comment]
> > The only reason I am looking at Steam's flatpak version is because of Gnome. There is no tray icon or entry in its "background app" in quick setting for official Steam application. I really hope either Gnome or Steam developers fix this issue.
> 
> It's not about the package, you need an extension called AppIndicator and KStatusNotifierItem Support. Gnome does not support tray icons in any version right now. You can get the extension from the Gnome website: https://extensions.gnome.org/extension/615/appindicator-support/ , or install it using the extension manager flatpak https://flathub.org/apps/com.mattjakeman.ExtensionManager , I recommend the latter.

That extension is broken on 45. i have tried it already, I think the author needs to update it to V.57 on the extension site. 
I already have created a bug report for it on their github as other people are also facing the same issue,  https://github.com/ubuntu/gnome-shell-extension-appindicator/issues/466#issuecomment-1750688870

I think Steam is working on to include XDG desktop portal on their native application, or at least that's the hint I got. idk if they will do this soon or not though, also created a report on that regard. https://github.com/ValveSoftware/steam-for-linux/issues/10105  

Cheers!! :))

**Comment by qoijjj** [comment]
> > The flatpak is not an official package by valve so it can't have the verified seal. Valve devs have helped get it working in the past when there were issues but unless they take over the package and make it official it can't have the verified seal/checkmark if I understand it correctly.
> > I think many of use are hoping that valve will adopt it and make the flatpak an officially supported version but so far that has not happened
> 
> But I've read that in order to get the "verified" seal, the app doesn't necessarily have to be packaged directly by the developer. It can also be verified if the developer approves a third party to package his app. For example, when it comes to very big projects like Libreoffice, it's obvious that it can't be packaged directly by the developers, so different maintainers are responsible for the various package formats. And it seems to me the maintainers who package the Libreoffice Flatpak aren't even affiliated with The Document Foundation, but Libreoffice is still verified in Flathub.
> 
> So I say maybe if someone will try to contact Valve and ask for Flathub verification approval, I think there is a chance it will accept that, especially if it helped with the Steam Flatpak at some point like you said.

That's correct. There are steps involved that the upstream developers must take, but they don't have to be the ones packaging the flatpak:
https://discourse.flathub.org/t/how-do-you-get-a-flathub-submission-verified/3714

There are many other flatpak apps on flathub that are verified this way.

**Comment by Hydranet** [comment]
> That extension is broken on 45.

I was using it on Gnome 45 and it was working for me. Since then I have updated to Gnome 46 and built the extension from the master branch and it's working on Gnome 46 as well. They haven't made an official Gnome 46 release yet because the extension is managed by Ubuntu who don't have Gnome 46 yet any of their current active Ubuntu versions.
